api/access/server.go

```diff
@@ -170,12 +173,66 @@ func (s *Server) ExecuteScriptAtBlockHeight(ctx context.Context, in *access.Exec
 	return nil, errors.New("not implemented")
 }
 
-func (s *Server) GetEventsForHeightRange(ctx context.Context, in *access.GetEventsForHeightRangeRequest) (*access.EventsResponse, error) {
-	return nil, errors.New("not implemented")
+func (s *Server) GetEventsForHeightRange(_ context.Context, in *access.GetEventsForHeightRangeRequest) (*access.EventsResponse, error) {
+	var events []*access.EventsResponse_Result
+	for height := in.StartHeight; height <= in.EndHeight; height++ {
+		ee, err := s.index.Events(height)
+		if err != nil {
+			return nil, fmt.Errorf("could not get events at height %d: %w", height, err)
```

Comment:
Personally, I still prefer my error syntax: `could not get events (height: %d): %w`, but OK, not a big deal.